docs/UserGuide.md

```diff
@@ -140,7 +140,7 @@ Deletes the specified flashcard from LingoGO!.
 Format: `delete INDEX`
 
 * Deletes the flashcard at the specified `INDEX`.
-* The index refers to the index number shown in the displayed flashcard list.
+* The index refers to the index number shown in the current list.
```

Comment:
i think it would be good to include minimally 'displayed'. the flashcards can be filtered by the user, but they still exist in the overall list

the original text was 'displayed person list'